en/starter/generator/core_concepts.md

```diff
@@ -253,4 +260,33 @@ This service uses the [SwiftMetrics package](https://github.com/RuntimeTools/Swi
 
 The connection details for this client are loaded by the configuration boilerplate code and are passed to the SwiftMetrics auto-scaling client.
 
+#### Watson Conversation IBM Cloud Service
+This service uses the [Watson Swift SDK package](https://github.com/watson-developer-cloud/swift-sdk), which allows Kitura applications to build Watson-powered applications, specifically in this case the IBM Watson Conversation service.
+
+With the IBM Watson Conversation service you can create cognitive agents--virtual agents that combine machine learning, natural language understanding, and integrated dialog scripting tools to build outstanding projects, such as a chat room with an integrated Watson chat bot.
```

Comment:
I don't understand the wording 'cognitive agents--virtual agents'
Do we need to use the word 'outstanding' in this context?